README.md

```diff
@@ -151,6 +151,10 @@ SEQREPO_LRU_CACHE_MAXSIZE sets the lru_cache maxsize for the sqlite
 query response caching. It defaults to 1 million but can also be set to
 "none" to be unlimited.
 
+SEQREPO_FD_CACHE_MAXSIZE sets the lru_cache size for file handler caching during FASTA sequence retrievals. 
+It defaults to 0 to disable any caching, but can be set to a specific value or "none" to be unlimited. Using 
+a moderate value (>10) will greatly increase performance of sequence retrieval.
+
```

Comment:
What is the downside of increasing this value?

Reply:
I think the only downside is in the potential for resource exhaustion on the system? But I'd defer to the discussion in https://github.com/biocommons/biocommons.seqrepo/issues/112 for additional information. Should I add a statement that states if the value is too large, there is a risk for exhaustion?